### PR TITLE
Bugfix for newline

### PR DESCRIPTION
**Pull request by tani**
Refer: #77 
Cheers!

### CODE REVIEW

File: src/ginger.js [hunk 1]
```diff
@@ -70,9 +70,10 @@ function reporter(context, options = {}) {
           .forEach((correction) => {
             const index = correction.start;
             const originalPosition = source.originalPositionFromIndex(index);
```
Comment (azu) [inline src/ginger.js]: Can we remove this?

and report error with `index`

```
new RuleError(message, { index: originalIndex })
```

Comment (tani) [reply src/ginger.js]: Sounds good. I updated.

**Review by azu — APPROVED** [review]
LGTM

**Comment by azu** [comment]
🚀 Release https://github.com/textlint-rule/textlint-rule-ginger/releases/tag/2.2.1

**Comment by tani** [comment]
Cool! Thank you for releasing immediately.

**Pull request merged**